test.js

```diff
@@ -0,0 +1,37 @@
+const assert = require('assert')
+const { describe, it } = require('mocha')
+const parser = require('./')
+
+describe('parser', () => {
+  describe('<type> ":" <text>', () => {
+    it('parses summary with no scope', () => {
+      const message = parser('fix: a really weird bug')
+      assert.strictEqual(message.type, 'message')
```

Comment:
We could simplify this if we can match on either a deep object or json string, in combination with [`unist-builder`](https://www.npmjs.com/package/unist-builder).

Reply:
I think at that point using snapshot testing [like in c8](https://github.com/bcoe/c8/blob/master/test/integration.js#L31) would perhaps be the most elegant, I expect the trees will be fairly readable.